### ISSUE TITLE
CLI for RPFM.

### ISSUE THREAD

**Issue by Frodo45127** [Frodo45127/rpfm#48]
Someone asked for this, and it's not a bad idea to make it work better with scripts and all that stuff, but needs some planning and investigation on how to do it.

**Comment by mklabs** [comment]
I highly +1 this. Having a CLI interface to RPFM could allow a lot of use case to interface RPFM with other applications.

PFM has a CLI part which allows you to interact with mod .pack, like adding a new file to a pack.

I personally would be super glad if RPFM could implement the same system, which would allow me to implement an idea I have for quite some time. Until now, I messed up with the Assembly Kit raw xml db files to do the same thing, but if RPFM could have a CLI to manipulate pack files, DB files and possibly loc files, that would be wonderful.

**Comment by Frodo45127** [comment]
Implemented in 2.0.

**Comment by edongashi** [comment]
@Frodo45127 do you mind adding some example args to the CLI help? I can't figure out how to export/import something from a packfile...

```
./rpfm_cli.exe -v -p mymod.pack -g warhammer_2 packfile -E db db
```

Trying the above fails with:

```
16:30:51 [INFO] Game Selected: warhammer_2
16:30:51 [INFO] Verbosity level: 1
16:30:51 [ERROR] Error while trying to read the following folder:
"db"
This means that path may not be readable by RPFM (permissions? other programs locking access to it?) or may not exists at all.
```
